frontend/packages/core/src/Feedback/tooltip.tsx

```diff
@@ -26,7 +26,7 @@ const StyledTooltip = styled(BaseTooltip)((props: { maxwidth?: string }) => ({
   },
 }));
 
-export interface TooltipProps extends Pick<MuiTooltipProps, "interactive" | "placement"> {
```

Comment:
is the interactive prop not available in muiv5?

Reply:
Interactive was removed as they're all interactive by default. Now the only interactive option is `disableInteractive`. Which probably makes sense to add on, let me fix.